obspy/taup/helper_classes.py

```diff
@@ -68,3 +68,57 @@ def __init__(self, topDepth=None, botDepth=None, ray_param=-1):
     'SplitLayerInfo',
     ['sMod', 'neededSplit', 'movedSample', 'ray_param']
 )
+
+
+class Arrival(object):
+    """
+    Convenience class for storing parameters associated with a phase arrival.
+
+    :ivar phase: :class:`~obspy.taup.seismic_phase.SeismicPhase` that
+        generated this arrival
+    :ivar distance: Actual distance in degrees
```

Comment:
The only annoyance is they appear as "Variables", not as "Attributes" like the two `@property` methods do.